packages/devextreme/js/common.d.ts

```diff
@@ -270,6 +270,24 @@ export type FirstDayOfWeek = 0 | 1 | 2 | 3 | 4 | 5 | 6;
  */
 export type Format = 'billions' | 'currency' | 'day' | 'decimal' | 'exponential' | 'fixedPoint' | 'largeNumber' | 'longDate' | 'longTime' | 'millions' | 'millisecond' | 'month' | 'monthAndDay' | 'monthAndYear' | 'percent' | 'quarter' | 'quarterAndYear' | 'shortDate' | 'shortTime' | 'thousands' | 'trillions' | 'year' | 'dayOfWeek' | 'hour' | 'longDateLongTime' | 'minute' | 'second' | 'shortDateShortTime';
 
+/**
+ * @public
+ * @namespace DevExpress.common
+ * @type object
+ */
```

Comment:
We did not mark the property (line 424) with any jsdoc tags because we don't want it to show up in the docs (as it's internal API)

So, it makes sense to avoid showing in the docs the type for this property.

Let's just remove the js doc tags from here?